src/db/utils/get-thread-by-id/get-thread-by-id.ts

```diff
@@ -0,0 +1,9 @@
+import { ThreadModel } from "../../../models/thread/thread.model";
+/**
+ * Finds user by ID and only returns relevant properties
+ * @param userId ObjectId
```

Comment:
For this doc string, should it be `threadId`?
```suggestion
 * @param threadId ObjectId
```

Reply:
:pray: